buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -178,51 +178,60 @@ public static int hashCode(ByteBuf buffer) {
 
     /**
      * Returns {@code true} if and only if the two specified buffers are
-     * identical to each other as described in {@code ChannelBuffer#equals(Object)}.
-     * This method is useful when implementing a new buffer type.
+     * identical to each other for {@code len} bytes starting at {@code iLhs}
```

Comment:
I don't know but I find the names for the params not very good. Any better ideas ?